documentation/release_notes.rst

```diff
@@ -8,6 +8,51 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.1.0
+===============
+
+New Features
+------------
+- Added ``generate``, ``generate_n``, ``transform`` algorithms to `Tested Standard C++ API`_.
+- Improved performance of the ``inclusive_scan``, ``exclusive_scan``, ``reduce`` and
+  ``max_element`` algorithms on heterogeneous devices.
+- Improved performance of ``sort`` and ``stable_sort`` APIs for arithmetic data types
+  used with DPC++ execution policies.
+
+Fixed Issues
+------------
+- Fixed ``TBB headers not found`` issue occurring with libstdc++ version 9 when
+  oneTBB headers are not present in the environment. The fix requires inclusion of
+  the oneDPL headers before the libstdc++ headers.
+- С++ 17 is now required as a minimal standard version when oneDPL is integrated through CMake
+  on Linux and Windows with some exceptions, see `New in This Release`_
```

Comment:
Should it be something like "oneDPL CMake scripts now enforce C++17 as the minimally required language version"? Is it important to mention Windows and Linux? If yes, asterisks (*) might be needed. And I would remove the "some exceptions" disclaimer.